pandas/io/parsers.py

```diff
@@ -3829,3 +3862,17 @@ def _validate_skipfooter(kwds: Dict[str, Any]) -> None:
             raise ValueError("'skipfooter' not supported for iteration")
         if kwds.get("nrows"):
             raise ValueError("'skipfooter' not supported with 'nrows'")
+
+
+def _can_close(original_handle, wrapped_handle) -> Tuple[bool, bool]:
+    """Check whether the output of io/common/get_handle can be closed."""
```

Comment:
Could change `get_handle` instead of having this function:
- add two more return values to `get_handle`
- and probably should use a named tuple then (`get_handle` would have four values)?

Reply:
I started with that. Merging `get_handle` and `get_filepath_or_buffer` would make sense!